src/generate/apiModel.ts

```diff
@@ -0,0 +1,139 @@
+/*
```

Comment:
I generated them from my IDE and then filled in the details.  What are the differents specifically here?

Reply:
There's a lot of JSDoc that doesn't need to exist in TSDoc because the type system already provides that information, and TSDoc is [more limited]( 
https://github.com/microsoft/tsdoc/blob/2a6bd2e49163ffe558a8a35224e79123624aa9ad/tsdoc/src/details/StandardTags.ts#L448-L473) (I couldn't find any actual doc pages with that list), and typedoc is [more so](https://typedoc.org/guides/doccomments/#supported-tags).

Specific things I noticed:
* `@param` / `@returns` shouldn't have type info or default values
* `@class` / `@export` / `@memberof` aren't valid TSDoc tags
* `@description` isn't a valid TSDoc tag and I think that including it will possibly result in "@description" appearing in the generated documentation.

Reply:
We talked about this in standup.  I will create a followup to bring the tsdoc linting rules into raml-toolkit to enforce it there as well.